src/renderer/render_api_opengles.cpp

```diff
@@ -1535,6 +1535,150 @@ class RHI_OpenGL : public TrRenderHardwareInterface
     if (TR_UNLIKELY(CheckError(req, reqContentRenderer) != GL_NO_ERROR || options.printsCall))
       PrintDebugInfo(req, nullptr, nullptr, options);
   }
+  TR_OPENGL_FUNC void OnVertexAttribI4i(VertexAttribI4iCommandBufferRequest *req,
+                                        renderer::TrContentRenderer *reqContentRenderer,
+                                        ApiCallOptions &options)
+  {
+    optional<GLint> loc = reqContentRenderer->getContextGL()->getAttribLoc(req);
+    if (loc == nullopt) [[unlikely]]
+    {
+      DEBUG(LOG_TAG_ERROR,
+            "vertexAttribI4i(): Failed to get location for attrib '%s' in program %d",
+            req->locationQueryName.c_str(),
+            req->program);
+      return;
+    }
+
+    glVertexAttribI4i(loc.value(), req->x, req->y, req->z, req->w);
+    if (TR_UNLIKELY(CheckError(req, reqContentRenderer) != GL_NO_ERROR || options.printsCall))
```

Comment:
```suggestion
    if (CheckError(req, reqContentRenderer) != GL_NO_ERROR || options.printsCall) [[unlikely]]
```
New added code should use `[[unlikely]]` or `[[likely]]`, please.